File: Solutions/605/python3/program.py
# Copy from Quera

n = int(input())
if n <= 2:
    print(n)
else:
    temp = [1, 2]
    for x in range(n - 2):
        temp.append((temp[-1] + temp[-2]) % (10 ** 9 + 7))
    print(temp[-1])

File: Solutions/605/java/program.java
// Copy from Quera

import java.util.Scanner;
public class Main {
    static Scanner s = new Scanner(System.in);
    public static void main(String[] args) {
        int n = s.nextInt();
        long F = 1, S = 2;
        for (int i = 0; i < n - 1; i++) {
            long mem = S + F;
            F = S % (long)(Math.pow(10, 9) + 7);
            S = mem;
        }
        System.out.println(F);
    }
}